tests/integration/test_lambda.py

```diff
@@ -1429,17 +1429,22 @@ def test_docker_command_for_separate_container_lambda_executor(self):
         handler = 'handler'
         lambda_cwd = '/app/lambda'
         network = 'compose_network'
+        dns = 'some-ip-address'
 
         config.LAMBDA_DOCKER_NETWORK = network
+        config.LAMBDA_DOCKER_DNS = dns
```

Comment:
Looks like the tests are failing because there's an assertion error in the lines below, and then the `config.LAMBDA_DOCKER_NETWORK` variable is not properly reset (unrelated to the changes in your PR).

Can we please wrap lines 1437-1444 in a `try`-`finally` block, so that the other tests are not affected?

```
config.LAMBDA_DOCKER_NETWORK = network
config.LAMBDA_DOCKER_DNS = dns
try:
    ...
finally:
    config.LAMBDA_DOCKER_NETWORK = ''
    config.LAMBDA_DOCKER_DNS = ''
```

Reply:
Added https://github.com/localstack/localstack/pull/3027/commits/37a3c1952c03ad997231ecd752621de892beb476
(squashed first)